packages/ketcher-core/src/application/formatters/formatterFactory.ts

```diff
@@ -70,46 +65,15 @@ export class FormatterFactory {
   create(
     format: SupportedFormat,
     options?: FormatterFactoryOptions,
-    queryPropertiesAreUsed?: boolean,
   ): StructFormatter {
-    const [molSerializerOptions, structServiceOptions] =
-      this.separateOptions(options);
+    const [, structServiceOptions] = this.separateOptions(options);
 
     let formatter: StructFormatter;
     switch (format) {
-      case SupportedFormat.ket:
+      case SupportedFormat.ket: {
         formatter = new KetFormatter(new KetSerializer());
         break;
-
-      case SupportedFormat.mol:
-        if (queryPropertiesAreUsed) {
-          formatter = new ServerFormatter(
-            this.#structService,
-            new KetSerializer(),
-            format,
-            structServiceOptions,
-          );
-        } else {
-          formatter = new MolfileV2000Formatter(
```

Comment:
the frontend has a dependency on the editor/ketcher instance which is currently broken for the case of multiple drawers on the same page.